assets/js/script.js

```diff
@@ -119,209 +102,214 @@ var randomQuestion;
 var currentQuestion;
 var questionCount = 0;
 
-var start = document.getElementById("start-btn");
-var secondsLeft = document.querySelector(".time");
-var container = document.querySelectorAll(".container");
-var scoreBoard = document.querySelector(".score");
-var questionNum = document.querySelector(".question-number");
-var questionEl = document.getElementById("question");
-var nextBtn = document.querySelector(".next-btn");
+var start = document.getElementById('start-btn');
```

Comment:
they probably haven't covered this, but beware global variables - wrapping this in a class/object is usually the best practice:

```
var QuestionGame = {}
QuestionGame.start = document.getElementById('start-btn')
...
```

This is likely just because this is an early class, and they will teach this later.

Reply:
Additionally consider naming elements with something that denotes they are a member of the DOM vs an action or item.

If I saw `var start` I wouldn't be sure if it's a function, the start time, or something else, whereas `var startBtn` is very clear.